test/SparqlAntlrParserTest.cpp

```diff
@@ -185,20 +185,19 @@ TEST(SparqlExpressionParser, First) {
   ASSERT_FLOAT_EQ(25.0, std::get<double>(result));
 }
 
-TEST(SparqlParser, ComplexConstructQuery) {
+TEST(SparqlParser, ComplexConstructTemplate) {
```

Comment:
If this is now only a test for the Construct template, also include one test for the complete `ConstructQuery` parsing.